.github/workflows/test-docker-compose.yaml

```
@@ -0,0 +1,167 @@
+name: Docker Compose Validation
+
+# Purpose of this workflow is to execute once the docker images have been built
+# to check if the compose file can spin up the indexer containers as well as they
+# are healthy. It depends on the indexer images workflows, so the first job is just
+# to make sure the "upstream" workflows have completed successfully
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
```

I think, we can do the following first so that it will fail fast if there is any syntactic issues first. we can save resource and time? :
```
validate-syntax: 
      name: Validate Docker Compose Syntax
      runs-on: ubuntu-latest
      steps:
        - uses: actions/checkout@v3
        - run: docker compose config --quiet
```

---

Yeah, I wasn't aware of this, I can add it